src/api_status.rs

```diff
@@ -43,11 +44,5 @@ pub mod tests {
     fn can_get_api_info() {
         let client = ShodanClient::new(get_test_api_key());
         let response = client.get_api_info().unwrap();
-
```

Comment:
Why are you removing this?

Reply:
There is nothing to assert. The error will be known during .unwrap().